skrf/network.py

```diff
@@ -1977,6 +1977,8 @@ def read_touchstone(self, filename: Union[str, TextIO],
         # set z0 before s so that y and z can be computed
         if touchstoneFile.is_from_hfss():
             self.gamma, self.z0 = touchstoneFile.get_gamma_z0()
+            # This is the s_def that HFSS uses
+            self.s_def = 'traveling'
```

Comment:
not necessarily, I think. There is an option in HFSS, as well as in ANSYS Circuit, to switch from a travelling to power wave definition (matching on conjugate match). I'll check that.

Reply:
I looked at HFSS options and didn't find a setting for it a while ago. It might be there somewhere though, but this should be the default.

Reply:
It's in the Circuit setting, but I never know if it affects Circuit only, or both Circuit and HFSS
![image](https://user-images.githubusercontent.com/4642848/169645598-1b00f91b-265d-41f2-b49e-e582b20d57c0.png)